generator/tree.go

```diff
@@ -302,6 +302,9 @@ func generateConfigModule(cfg *ModuleConfig, node *Node, nameToNode map[string]*
 	// Remove redundant OIDs to be walked.
 	toWalk := []string{}
 	for _, oid := range cfg.Walk {
+		if strings.HasPrefix(oid, ".") {
+			logger.Error("OID starts with a '.'", "OID", oid)
```

Comment:
We prefer "fail fast" in order to highlight invalid configurations. Let's just error out the whole thing.

```suggestion
			return nil, fmt.Errorf("invalid OID %s, prefix of '.' should be removed", oid)
```

Reply:
Thanks for the feedback. I have updated it as you suggested. I also updated the commit message.